spec/models/room_spec.rb

```diff
@@ -3,5 +3,32 @@
 RSpec.describe Room, type: :model do
   describe "relationships" do
     it { should belong_to :hotel }
+    it { should have_many :guests }
```

Comment:
There should be 2 relationship tests here: `it { should have_many :guest_rooms}` AND `it { should have_many(:guests).through(:guest_rooms) }`